serf/snapshot.go

```diff
@@ -320,7 +320,16 @@ func (s *Snapshotter) appendLine(l string) error {
 
 	n, err := s.buffered.WriteString(l)
 	if err != nil {
-		return err
+		wrErr := err
```

Comment:
I thought about that, but figured it should fail fast if its low on disk or other resources. Seems valid to only do this every 30 seconds or so though, so I'll fix this.